src/bsim/winter2021/Bacterium.java

```diff
@@ -218,6 +224,8 @@ public Bacterium divide() {
 
         this.parent_id = this.id;
         this.lifetime = 0;
+        this.toxintime = 0;      
+        this.contact= 0;              

```

Comment:
Remove these from Bacterium.java; these should only be associated with certain types of bacteria